### PR TITLE
FIX: Fixed layout for default screen resolution

### PR DESCRIPTION
**Pull request by kotesh-arya**
- ### Updated website layout suitable for default  screen resolutions

### CODE REVIEW

File: Components/Cart/cart.html [hunk 1]
```diff
@@ -13,70 +13,78 @@
...
+                    </div>
+                    <div class="card-content price">
+                        <h2>Rs.500 <strike><span class="light-text">Rs.2000</span></strike> </h2>
+                        <h3>(75% off)</h3>
+                    </div>
+                    <div class="quantity-setting">
+                        <h4>Quantity:</h4>
+                        <div class="counter-buttons">
                             <button class="btn-counter">-</button>
                             <input class="quantity-input" type="text">
```
Comment (TejasShekar) [inline Components/Cart/cart.html]: type is number and set a default value to it.

File: Components/Cart/cart.css [hunk 2]
```diff
@@ -71,30 +64,52 @@ body {
...
     display: flex;
     flex-direction: column;
     align-items: left;
     justify-content: space-between;
-    width: 35%;
-    margin: 0 2rem 4.5rem 2rem;
-    padding: 2rem 0 2rem 0;
+    width: 25rem;
+    margin: 0rem 2rem 0rem 2rem;
+    padding: 2rem 0 0rem 0;
```
Comment (Sharmaryan) [inline Components/Cart/cart.css:112]: why not to use shorthand  for margin here, ```  margin: 0 2rem; ```

File: Components/Cart/cart.html [hunk 3]
```diff
@@ -13,70 +13,78 @@
 
 <body>
     <div class="navigation-container">
-        <div class="logo-input">
+        <div class="">
```
Comment (Sharmaryan) [inline Components/Cart/cart.html:16]: this class is empty, either remove it or add class in it

File: Components/Cart/cart.html [hunk 4]
```diff
@@ -13,70 +13,78 @@
...
             <div class="nav-header">
-                <h1>Daily Fashion</h1>
-            </div>
-            <div class="search-input-container"><input class="header-input" type="search">
-                <img class=" input-icon" src="https://img.icons8.com/material/28/000000/search--v1.png" />
+                <a class="site-title" href="/Components/Homepage/homepage.html">
+                    <h1>DAILY FASHION</h1>
```
Comment (Sharmaryan) [inline Components/Cart/cart.html:19]: Since we are using react, we should use className  insted of class

File: Components/Homepage/homepage.html [hunk 5]
```diff
@@ -8,47 +9,65 @@
     <link rel="stylesheet" href="homepage.css">
     <title>Daily Fashion : Home</title>
 </head>
+
 <body>
     <div class="navigation-container">
-        <div class="logo-input">
+        <div class="">
             <div class="nav-header">
```
Comment (Sharmaryan) [inline Components/Homepage/homepage.html:16]: empty class

File: Components/LogIn/login.html [hunk 6]
```diff
@@ -39,8 +40,9 @@ <h1 class="header-large">LOG IN</h1>
             </div>
             <a class="forgot-password" href="">Forgot Password</a>
         </div>
```
Comment (Sharmaryan) [inline Components/LogIn/login.html:42]: you can put href='#' as of now

File: Components/ProductListing-filters/productlist.css [hunk 7]
```diff
@@ -21,18 +22,27 @@ body {
...
+}
+
+.header-icon {
+    color: var(--fa-primary-color);
+    font-size: 1.5rem;
 }
 
 .basic {
     border: 0px;
 }
```
Comment (Sharmaryan) [inline Components/ProductListing-filters/productlist.css:40]: you can remove px

File: Components/ProductListing-filters/productlist.css [hunk 8]
```diff
@@ -123,13 +130,14 @@ body {
     display: flex;
     flex-direction: column;
     justify-content: space-between;
-    padding: 0.6rem 3rem;
+    padding: 0.6rem 1rem;
+    margin-left: 0.5rem;
```
Comment (Sharmaryan) [inline Components/ProductListing-filters/productlist.css:134]: for values lesser than 1rem you can use px

**Review by Sharmaryan — APPROVED** [review]
Suggested few changes, you can make them, one thing I noticed you could use margin and padding's shorthand property

**Review by TejasShekar — APPROVED** [review]
- Categories are not clickable on the homepage.
- Multiple radio buttons can be selected at the same time. Fix it.

File: Components/Cart/cart.html (hunk 4 above)
Comment (kotesh-arya) [reply Components/Cart/cart.html:19]: No No... this is a Static project with HTML & CSS only 🙂

File: Components/Cart/cart.css (hunk 2 above)
Comment (kotesh-arya) [reply Components/Cart/cart.css:112]: Sure 👍🏻

File: Components/Homepage/homepage.html (hunk 5 above)
Comment (kotesh-arya) [reply Components/Homepage/homepage.html:16]: Updated...

File: Components/LogIn/login.html (hunk 6 above)
Comment (kotesh-arya) [reply Components/LogIn/login.html:42]: Yep, Done it

File: Components/ProductListing-filters/productlist.css (hunk 7 above)
Comment (kotesh-arya) [reply Components/ProductListing-filters/productlist.css:40]: Done 👍🏻

File: Components/ProductListing-filters/productlist.css (hunk 8 above)
Comment (kotesh-arya) [reply Components/ProductListing-filters/productlist.css:134]: To maintain consistency with the other value in that shorth-hand pair I have used rem

File: Components/Cart/cart.html (hunk 1 above)
Comment (kotesh-arya) [reply Components/Cart/cart.html]: Updated that, Thanks for pointing out it  👍🏻

**Pull request merged**